README.md

```diff
@@ -35,6 +35,26 @@ $ curl -LO https://github.com/ipinfo/cli/releases/download/ipinfo-1.1.0/ipinfo_1
 $ sudo dpkg -i ipinfo_1.1.0.deb
 ```
 
+### Windows via PowerShell
+
+Here's how to download and unpack the ipinfo CLI tool on Windows using PowerShell:
+
+```powershell              
+# Get info about the latest release from GitHUb 
+$latestRelease = (Invoke-RestMethod https://github.com/ipinfo/cli/releases/latest -Headers @{"Accept"="application/json" } -UseBasicParsing).tag_name 
+
+# Parse the version number
+$version=($latestRelease -split '-')[1]
+
+# build the filename for the Zip archive
+$zipFileName = "ipinfo_$($version)_windows_amd64.zip"
+
+Invoke-WebRequest -Uri "https://github.com/ipinfo/cli/releases/download/ipinfo-$version/$zipFileName" -UseBasicParsing -OutFile ./$zipFileName
+Unblock-File ./$zipFileName
+Expand-Archive -Path ./$zipFileName  -DestinationPath $env:LOCALAPPDATA\ipinfo -Force
+Remove-Item -Path ./$zipFileName
+```
```

Comment:
Is it possible to wrap this script into something similar to the [MacOS](https://github.com/ipinfo/cli/blob/master/ipinfo/macos.sh) / [Ubuntu/Debian](https://github.com/ipinfo/cli/blob/master/ipinfo/deb.sh) scripts? So that someone could do the windows version of this and have the script automatically run in almost a 1-liner:

```bash
# this would download a script containing all the above steps, run the script, and then delete itself.
$ curl -Ls https://github.com/ipinfo/cli/releases/download/ipinfo-1.1.0/macos.sh | sh
```

FYI the existing scripts are found in [this folder](https://github.com/ipinfo/cli/tree/master/ipinfo) and we include them in the final releases [here](https://github.com/ipinfo/cli/blob/master/ipinfo/release.sh#L38-L46).

Reply:
Yes, just put the statements in a file with extension .ps1 and then it could be handled in the same way as the curl example above. You can use PowerShell simething like this:
```
Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://github.com/ipinfo/cli/releases/download/ipinfo-1.1.0/setup-win.ps1')
```
This line would download the script and then launch it. You can see an example how it is used over at [scoop.sh](https://scoop.sh/)

Reply:
Ah ok, thank you!

We ended up already adding scripts @ https://github.com/ipinfo/cli/pull/80 so we don't actually need this anymore. Let us know though how those scripts work for you. Example: https://github.com/ipinfo/cli#windows-powershell